### ISSUE TITLE
imports don't work

### ISSUE THREAD

**Issue by mikofski** [NREL/bifacialvf#15]
when I try to run bifacial vf I get an import error:
```python
import bifacialvf
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-7112b735b939> in <module>
----> 1 import bifacialvf

~\Projects\bifacialvf\bifacialvf\__init__.py in <module>
----> 1 from bifacialvf import simulate  # main program
      2 from vf import getBackSurfaceIrradiances, getFrontSurfaceIrradiances, getGroundShadeFactors  # main subroutines
      3 from vf import getSkyConfigurationFactors, trackingBFvaluescalculator, rowSpacing # helper functions
      4 from sun import hrSolarPos, perezComp, solarPos, sunIncident # solar position and value
      5 from loadVFresults import loadVFresults # utility for reading result files

ImportError: cannot import name 'simulate'
```

**Comment by mikofski** [comment]
Hi @shirubana and @cdeline , I think this is ready to merge, and it's prerequisite for #13 to clean up the docstrings and generate some Sphinx documentation and #14 to add some testing - thanks!

**Comment by mikofski** [comment]
closed by #13 (f8664273cdbb2c80a4e751731ebd04abda2dd9d0)
